src/bin/nydus-image/main.rs

```diff
@@ -544,8 +554,13 @@ fn init_log(matches: &ArgMatches) -> Result<()> {
 
     // Safe to unwrap because it has a default value and possible values are defined.
     let level = matches.value_of("log-level").unwrap().parse().unwrap();
+    let rotate_size = matches
+        .value_of("log-rotate-size")
+        .unwrap()
+        .parse::<u64>()
+        .unwrap();
 
-    setup_logging(log_file, level).context("failed to setup logging")
+    setup_logging(log_file, level, rotate_size).context("failed to setup logging")
```

Comment:
rotate should be a verb while rotation is a noun. So better to rename it to rotation_size

Reply:
You are right, i have renamed all "rotate".